packages/lesswrong/components/votes/CommentsVote.jsx

```diff
@@ -50,7 +51,7 @@ class CommentsVote extends PureComponent {
     )
 
     return (
-      <div className={classNames("comments-item-vote"), classes.vote}>
+      <div className={classNames("comments-item-vote", classes.vote)}>
```

Comment:
Possibly breaks styling, by fixing only one out of a cancelling pair of bugs. As written, this applied the `classes.vote` class and not the `comments-item-vote` class. But also, `comments-item-vote` has styles which likely aren't wanted anymore. Better to delete the `.comments-item-vote` style from `packages/lesswrong/styles/_comments.scss` and also delete the `classNames` bit here.